staff/adria-arquimbau/mucho-layout/with BEM/index.html

```diff
@@ -0,0 +1,358 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Mucho/Layout</title>
+    <script src="https://kit.fontawesome.com/3bf2c7b64d.js"></script>
+    <link rel="stylesheet" href="output.css">
+</head>
+    <body>
```

Comment:
You have to review the naming of elements/blocks in BEM. Especially check the modifiers.
I'll give you some examples of correct names, so they can guide you...